src/views/templates/list/components/VirtualMachineTemplatesRow.tsx

```diff
@@ -42,7 +44,7 @@ const VirtualMachineTemplatesRow: React.FC<
         <ResourceLink kind="Namespace" name={obj.metadata.namespace} />
       </TableData>
       <TableData id="workload" activeColumnIDs={activeColumnIDs} className="pf-m-width-10">
-        {useWorkloadProfile(obj)}
+        {useWorkloadProfile(obj, t) || t('Other')}
```

Comment:
```suggestion
         { t(workloadLabels[workload] || 'Other') }
```
you whould also need to get the workload key `const workload = getTemplateWorkload(obj);`

Reply:
maybe a helper here may be nice ?
``` ts
const getWorkloadLabel = (workload: WorkloadProfileKeys): WorkloadProfileDisplayNames  => 
  workloadLabels[workload] || 'Other';
```

Reply:
Adding a helper here is not necessary. But let me think again about how the translation is implemented here. Thanks!